charts/victoria-metrics-k8s-stack/values.yaml

```diff
@@ -737,6 +737,31 @@ kubeControllerManager:
             caFile: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
             serverName: kubernetes
 
+## Component scraping kubeDns. Use either this or coreDns
+##
+kubeDns:
+  enabled: false
+  service:
+    enabled: false
+    dnsmasq:
+      port: 10054
+      targetPort: 10054
+    skydns:
+      port: 10055
+      targetPort: 10055
+    # selector:
+    #   k8s-app: kube-dns
```

Comment:
should selector be uncommenter and be empty?

Reply:
yep, it also can be non-empty - with default value.